src/ir/ty.rs

```diff
@@ -612,7 +612,7 @@ impl Type {
                             TypeKind::TemplateAlias(inner.unwrap(), args)
                         }
                         CXCursor_TemplateRef => {
-                            let referenced = location.referenced();
+                            let referenced = location.referenced().unwrap_or(location);
```

Comment:
This should probably [`expect`](https://doc.rust-lang.org/std/option/enum.Option.html#method.expect) the method to return a `Cursor`, so that we'll know when this doesn't return one